js/binance.js

```diff
@@ -795,6 +795,8 @@ module.exports = class binance extends Exchange {
         if (code >= 400) {
             if (code == 418)
                 throw new DDoSProtection (this.id + ' ' + code.toString () + ' ' + reason + ' ' + body);
+            if (body.indexOf ('Price * QTY is zero or less'))
```

Comment:
It lacks >= 0 in this conditional... still have to have it, unfortunately.

Reply:
Oh! Linter said nothing, will try to find the rule.

Reply:
No luck with the linter (

Reply:
No worries, I'll look into it, maybe we could make it transpile these conditionals without the `>= 0` clause. Thanks anyway for editing it!

Reply:
> maybe we could make it transpile these conditionals without the `>= 0` clause

No, that's definitely my fault, we need `=` part of comparison anyway it's good that you spot it.